package.json

```diff
@@ -24,11 +24,10 @@
   "dependencies": {
     "@actions/core": "^1.6.0",
     "@actions/github": "^5.0.0",
-    "axios": "^0.24.0",
-    "install": "^0.13.0",
-    "npm": "^8.1.2"
+    "axios": "^0.24.0"
   },
   "devDependencies": {
+    "@vercel/ncc": "^0.33.3",
```

Comment:
Added ncc as a dev dependency so things just work out the box without having to install it globally